### ISSUE TITLE
Too many indent guides

### ISSUE THREAD

**Issue by m0nkey-br4in** [haad/language-Ansible#11]
For some lines indent guides are drawn after each space. It looks like this:

![atom-too-many-indent-guides-example](https://cloud.githubusercontent.com/assets/8735883/18092138/d4086d04-6ed3-11e6-81af-b9b0726a3655.png)

Indents still work as intended for keyboard navigation though, i.e. the cursor jumps two spaces

**Comment by xenithorb** [comment]
I am also seeing weirdness like this: 

![image](https://cloud.githubusercontent.com/assets/10273878/18414667/6a0ee3d4-77a1-11e6-98a9-8802461eb2c4.png)

**Comment by haad** [comment]
hi can you share gist/code with me so I can try to reproduce it as I can't do it now.

**Comment by m0nkey-br4in** [comment]
A gist with what's on the screenshot? Sure, here you go: https://gist.github.com/m0nkey-br4in/ebd06a6ca9a0970c0f3e0e87ad684e03

Atom is version 1.10.2, language-ansible is version 0.1.11
Tabs are 2 spaces wide, I guess it's the default

**Comment by dev169** [comment]
I'm seeing this behavior as well. Atom `1.13.0-beta8`, latest `language-ansible` (`0.1.11`).

With tabs 4 spaces wide:

![image](https://cloud.githubusercontent.com/assets/5874526/21110919/e6f67534-c0a7-11e6-8f4e-abc433c78a8f.png)

And with tabs 2 spaces wide:

![image](https://cloud.githubusercontent.com/assets/5874526/21111053/6b3cba88-c0a8-11e6-81ca-afa20f7c2e3c.png)

**Comment by melck** [comment]
I'm not sure but i fix this issue by doing that :
```diff
From afe3dd792460e2b1315e1d06ede71e0fada3efe6 Mon Sep 17 00:00:00 2001
From: Melck  <m3lchior@gmail.com>
Date: Sat, 14 Jan 2017 14:10:39 +0100
Subject: [PATCH] fix #11, multiple indent guide

---
 grammars/ansible.cson | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grammars/ansible.cson b/grammars/ansible.cson
index a115441..f080313 100644
--- a/grammars/ansible.cson
+++ b/grammars/ansible.cson
@@ -34,7 +34,7 @@ patterns: [
     captures:
       "2":
         name: "keyword.other.ansible"
-    match: "(\\s*(\\w+:))?\\s"
+    match: "(\\s*(\\w+:))\\s"
     name: "text"
   }
   {
-- 
2.9.3
```

If someone want to create pull request, i don't have time to do it.
